Src/WitsmlExplorer.Frontend/components/ContentView.tsx

```diff
@@ -81,6 +83,8 @@ const ContentView = (): React.ReactElement => {
         setObjectView(true);
       } else if (currentSelected === selectedLogTypeGroup) {
         setView(<LogsListView />);
+      } else if (currentSelected === displayGraph) {
+        setView(<LogsGraph />);
```

Comment:
Is there a reason for using NavigationContext for switching to LogsGraph? Would it not be simpler to keep the `displayGraph` variable in LogsListView and do something like `return displayGraph ? <ContentTable/> : <LogsGraph logs={logs}/>;`

Reply:
See CurveValuesView in https://github.com/equinor/witsml-explorer/pull/1995/files as an example.

Reply:
I followed your instruction by creating a new navigation type NavigationType.SelectLogTypeGraph. For me it seems to more logical approach.

Reply:
implemented, but needed to delete the new action and button